dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/api/RequestApi.kt

```diff
@@ -85,11 +89,106 @@ interface RequestApi {
         ],
     )
     @GetMapping(
+        value = ["/aggregated"],
         produces = ["application/json"],
     )
     @PreAuthorize("hasRole('ROLE_USER')")
     fun getAggregatedDataRequests(
         @RequestParam identifierValue: String? = null,
         @RequestParam dataTypes: Set<DataTypeEnum>? = null,
-    ): List<AggregatedDataRequest>
+        @RequestParam reportingPeriod: String? = null,
+    ): ResponseEntity<List<AggregatedDataRequest>>
+
+    /**
+     * A method to post a single request to Dataland.
+     * @param singleDataRequest includes necessary info for the single request
+     * @return response after posting a single data request to Dataland
+     *
+     */
+    @Operation(
+        summary = "Send a single request",
+        description = "A single of data requests for specific frameworks and companies is being sent.",
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Successfully processed a single data request."),
+        ],
+    )
+    @PostMapping(
+        value = ["/single"],
+        consumes = ["application/json"],
+        produces = ["application/json"],
+    )
+    @PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
Single Requests should be for now only possible for Uploaders

Reply:
Done

Reply:
ok